tools/rna_tools/trna_mod/gatk_wrapper.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="tRNAmodgatk" name="gatk_for_tRNAmod" version="0.1.1">
+  <description>GATK for tRNAmod analysis</description>
+  <requirements>
+	<requirement type="package" version="2.7">python</requirement>
+	<requirement type="package" version="1.4.1">samtools</requirement>
+	<requirement type="package" version="2.9.2">picard</requirement>
+	<requirement type="package" version="8.0.92">java-jdk</requirement>
+  </requirements>
+  <command>
+	<![CDATA[
+			 ##cleanup
+			 rm -f "${reference}.dict";
+			 rm -f "${reference}.fasta.fai";
+			 rm -f "${input}.bam.bai";
+			 rm -f "${input}.bai";
+			 ## prepare fasta index and dict and link files for correct file type
+             ln -sf "${input}" "${input}.bam";
+             ln -sf "${reference}" "${reference}.fasta";
+			 #if str($gatktool.tool) == "IndelRealigner":
+			     ln -sf "${intervals}" "${intervals}.intervals";
+			 #end if
+			 samtools faidx "${reference}.fasta" ;
+			 picard CreateSequenceDictionary R= "${reference}.fasta" O= "${reference}.dict";
+			 samtools index "${input}.bam";
+			 picard BuildBamIndex I="${input}.bam" O="${input}.bam.bai";
+			 			 
+			 ##now run gatk
+			 java -jar $__tool_directory__/GenomeAnalysisTK.jar
```

Comment:
There is now a conda package for GATK4 ... the problem with older versions and this PR is that we are not allowed to distribute the JAR file ... the license does not allow this for GATK < 4 :( sorry.